extension/content_script.js

```diff
@@ -317,8 +317,18 @@ function stopFadeTracker() {
 	let keywords_el = $("."+keywordClass)
 	let sentence_color = sentence_el.css('backgroundColor');
 	let keywords_color = keywords_el.css('backgroundColor');
-	sentence_el.stop().animate({'background-color': sentence_color});
-	keywords_el.stop().animate({'background-color': keywords_color});
+	sentence_el.stop().animate({'background-color': sentence_color},
```

Comment:
I don't understand this, but since we're at crunch time and I assume you tested this, can you just leave a TODO: for now so I can take a look later?

My q is: why can't just .stop()?

Reply:
Short answer: The .stop() only catches it when it's too late; it is necessary to reanimate it to full opacity in order for it not to be effected